### PR TITLE
[22088] Ensure default device theme is used on Android datePicker

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

**Comment by livecodepanos** [comment]
WIP until this is tested on older Android devices

File: engine/src/java/com/runrev/android/DialogModule.java [hunk 1]
```diff
@@ -149,7 +149,7 @@ public void onDismiss(DialogInterface p_dialog)
         Calendar t_calendar = Calendar.getInstance();
         t_calendar.setTimeInMillis(p_current * 1000);
         
-        DatePickerDialog t_dialog = new DatePickerDialog(m_engine.getContext(), t_date_listener,
+        DatePickerDialog t_dialog = new DatePickerDialog(m_engine.getContext(), android.R.style.Theme_DeviceDefault_Light_Dialog_Alert, t_date_listener,
```
Comment (montegoulding) [inline engine/src/java/com/runrev/android/DialogModule.java]: Cool @livecodepanos . Is there a reason not to use `Theme_DeviceDefault_Dialog_Alert` as in do we need to force `Light`?

Comment (livecodepanos) [reply engine/src/java/com/runrev/android/DialogModule.java]: Hehe, the reason for that was that I did not notice there was such an option :)

Will update the PR asap

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 29320d2

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 29320d220ec53da1aeaedb7649e933e27aeecf52

**Comment by livecode-vulcan** [comment]
😎 test success 29320d220ec53da1aeaedb7649e933e27aeecf52

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/572)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/564)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/564)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/565)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/568)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2091)
* try-community-universal-ios-iphoneos12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.1/builds/313)
* try-community-universal-ios-iphonesimulator12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.1/builds/313)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2117)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2107)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2106)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2140)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2080)

**Pull request merged**